src/test/resources/application.properties

```diff
@@ -3,3 +3,6 @@ spring.datasource.driverClassName=org.h2.Driver
 spring.datasource.username=root
 spring.datasource.password=Kolnerievgenii02//04
 spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+
+jwt.expiration=300000
+jwt.secret=i-am-a-spy-token-give-you-opportunity-to-take-some-information
```

Comment:
It’s funny) but it’s a project for your CV so better use a more secure jwt.secret